build/includes/google-cloud.mk

```diff
@@ -48,23 +47,52 @@ terraform-clean:
 	rm -r ./.terraform
 	rm ./terraform.tfstate*
 
-
+# Creates a cluster and install release version of Agones controller
+# Version could be specified by AGONES_VERSION
 gcloud-terraform-cluster: GCP_CLUSTER_LEGACYABAC ?= false
 gcloud-terraform-cluster: GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT ?= 4
 gcloud-terraform-cluster: GCP_CLUSTER_NODEPOOL_MACHINETYPE ?= n1-standard-4
+gcloud-terraform-cluster: AGONES_VERSION ?= 0.9.0
 gcloud-terraform-cluster: $(ensure-build-image)
 gcloud-terraform-cluster:
 ifndef GCP_PROJECT
 	$(eval GCP_PROJECT=$(shell sh -c "gcloud config get-value project 2> /dev/null"))
 endif
-	$(DOCKER_RUN) bash -c 'export TF_VAR_password=$(GKE_PASSWORD) && \
-		cd $(mount_path)/build && terraform apply -auto-approve  \
+	$(DOCKER_RUN) bash -c 'export TF_VAR_agones_version=$(AGONES_VERSION) && \
+	    export TF_VAR_password=$(GKE_PASSWORD) && \
+		cd $(mount_path)/build && terraform apply -auto-approve -var values_file="" \
+		-var chart="agones" \
 	 	-var "cluster={name=\"$(GCP_CLUSTER_NAME)\", machineType=\"$(GCP_CLUSTER_NODEPOOL_MACHINETYPE)\", \
 		 zone=\"$(GCP_CLUSTER_ZONE)\", project=\"$(GCP_PROJECT)\", \
 		 initialNodeCount=\"$(GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT)\", \
 		 legacyABAC=\"$(GCP_CLUSTER_LEGACYABAC)\"}"'
 	$(MAKE) gcloud-auth-cluster
-	$(MAKE) setup-test-cluster
+
+# Creates a cluster and install current version of Agones controller
+# Set all necessary variables as `make install` does
+gcloud-terraform-install: GCP_CLUSTER_LEGACYABAC ?= false
+gcloud-terraform-install: GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT ?= 4
+gcloud-terraform-install: GCP_CLUSTER_NODEPOOL_MACHINETYPE ?= n1-standard-4
+gcloud-terraform-install: ALWAYS_PULL_SIDECAR := true
+gcloud-terraform-install: IMAGE_PULL_POLICY := "Always"
```

Comment:
Given that we're installing a production version of Agones - probably don't need to be always pulling images, wdyt?

Reply:
This make target will install not production, but development version as oppose to `make gcloud-terraform-cluster` (install production versions)